frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -451,7 +451,7 @@
 
         on-context-menu
         (mf/use-fn
-         (mf/deps component-id)
+         (mf/deps on-context-menu component-id)
```

Comment:
This fixes the selection issue, but the context menu is not ok. I still see all options for components (like duplicate main or see main component).

I'm not very sure of what needs to be the desired behavior. For example, what happens if I do this and then choose "Group"?

Reply:
This fix the reported issue, maybe you can create a new issue and fix it in another MR.

Reply:
This is in the issue:

  Desired result
  
  - Multiselection is keeped
  - The options available are: Delete and Group